### PR TITLE
feat(ct-metrics): add initial conntrack metrics for Prometheus

### PR DESCRIPTION
**Pull request by SRodi**
# Description

Create initial metrics for conntrack:
- packets_count_per_connection [gauge] see example below
- bytes_count_per_connection [gauge]
- connections_count [count] **WIP**

## Related Issue

#806 

## Checklist

- [x] I have read the [contributing documentation](https://retina.sh/docs/contributing).
- [x] I signed and signed-off the commits (`git commit -S -s ...`). See [this documentation](https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification) on signing commits.
- [x] I have correctly attributed the author(s) of the code.
- [x] I have tested the changes locally.
- [x] I have followed the project's style guidelines.
- [x] I have updated the documentation, if necessary.
- [x] I have added tests, if applicable.

## Screenshots (if applicable) or Testing Completed

This example shows a long-lived TCP connection between client `src_ip="10.244.0..230` and server `10.244.1.85` 

image

This example shows the `bytes_count_per_connection` where `dst_ip=10.244.0.113` which is the pod IP for `metrics-server` on one of the cluster nodes. 

image

(WIP) Here is some initial work on Hubble metrics

image

## Considerations

Metrics have been added as `pod-level metrics` since the number of time series increases overtime based on number of new connections in the cluster - the count of time series is the same for both `packets_count_per_connection` and `bytes_count_per_connection` metrics since both use the same labels.

image

---

Please refer to the [CONTRIBUTING.md](../CONTRIBUTING.md) file for more information on how to contribute to this project.

### CODE REVIEW

**Comment by MikeZappa87** [comment]
Did we think though around scalability, performance and stability when adding these metrics? Do we have any concerns? Do we have numbers to back this up?

File: pkg/metrics/metrics.go [hunk 1]
```diff
@@ -140,6 +140,26 @@ func InitializeMetrics() {
 		dnsResponseCounterDescription,
 	)
 
+	// Conntrack Metrics
+	ConntrackPacketsCounter = exporter.CreatePrometheusGaugeVecForMetric(
```
Comment (vakalapa) [inline pkg/metrics/metrics.go:144]: Can you find a way to integrate these into Hubble metrics as well ?

Comment (SRodi) [reply pkg/metrics/metrics.go:144]: still WIP, but some initial work is done, see this screenshot
image

**Comment by SRodi** [comment]
after an offline discussion with @vakalapa and @nddq it was agreed to abandon this PR in favor of multiple PRs to break down the solution into:
* data plane implementation
* control plane Hubble
* control plane legacy
* advance metrics

It was also agreed to disable those metrics by default, with the use of a feature flag similar to `enableConntrackMetrics` 

cc @MikeZappa87

**Pull request closed without merging**